index.html

```diff
@@ -174,7 +174,7 @@ <h2 class="wow fadeInLeftBig">Our Services</h2>
                     <div class="service_iconarea">
                       <span class="fa fa-paper-plane service_icon"></span>
                     </div>
-                    <h3 class="service_title">HACKATHON - InOut</h3>
+                    <a href="https://hackinout.co/" target="blank"><h3 class="service_title">HACKATHON - InOut</h3></a>

```

Comment:
It should be like this :- 
target="_blank"
Notice the underscore before the word blank.

So, change it and commit. And now you will be with two commits in the bug-links branch. You will have to merge those two commits and again push the code.
Please ping if you are not sure how to do it !